### PR TITLE
Refs #12570 -  Hammer commands for repo and CVV export

### PR DESCRIPTION
**Pull request by beav**
This adds two commands to hammer which support repo and CVV export.

### CODE REVIEW

File: lib/hammer_cli_katello/content_view_version.rb [hunk 1]
```diff
@@ -182,6 +182,17 @@ def request_params
...
 
+    class ExportCommand < HammerCLIKatello::Command
+      action :export
+      command_name "export"
+
+      success_message _("Export complete")
+      failure_message _("Could not export")
+
+      build_options
+
```
Comment (komidore64) [inline lib/hammer_cli_katello/content_view_version.rb]: kill this blank line

File: lib/hammer_cli_katello/repository.rb [hunk 2]
```diff
@@ -290,6 +290,17 @@ def request_params
       build_options :without => ["uuids"]
     end
 
+    class ExportCommand < HammerCLIKatello::Command
+
```
Comment (komidore64) [inline lib/hammer_cli_katello/repository.rb]: kill this blank line

File: lib/hammer_cli_katello/repository.rb [hunk 3]
```diff
@@ -290,6 +290,17 @@ def request_params
...
+      resource :repositories, :export
```
Comment (komidore64) [inline lib/hammer_cli_katello/repository.rb]: the resource is already specified above, so you can just say ``action :export``.

**Comment by komidore64** [comment]
This PR needs to point to a redmine issue

**Comment by komidore64** [comment]
ACK pending comments

File: lib/hammer_cli_katello/repository.rb (hunk 3 above)
Comment (beav) [inline lib/hammer_cli_katello/repository.rb]: I am not sure why, but it doesn't work without `:repositories`. The other classes in this file have it as well.

**Comment by beav** [comment]
@komidore64 PR updated for whitespace issues and RM number added

File: lib/hammer_cli_katello/repository.rb (hunk 3 above)
Comment (komidore64) [inline lib/hammer_cli_katello/repository.rb]: okay, that's cool.

**Comment by beav** [comment]
@komidore64 https://projects.theforeman.org/issues/13636 for the 'version' issue

**Pull request merged**